addons/point_of_sale/tests/test_anglo_saxon.py

```diff
@@ -57,6 +56,7 @@ def setUp(self):
         self.pos_config.write({'payment_method_ids': [(6, 0, self.cash_payment_method.ids)]})
 
 
+@odoo.tests.tagged('post_install', '-at_install')
```

Comment:
@pimodoo pardon the ping. But what do you think about this? I added it because I think there is no point for this test to run at_install (before post_install) because we want to have chart template before launching a pos.session. And only at post_install that the chart template is installed.

Reply:
@caburj It is good to, they should all be pos install